sound/mpd/files/mpd.init

```diff
@@ -9,21 +9,32 @@ PROG=/usr/bin/mpd
 CONFIGFILE=/etc/mpd.conf
 NICEPRIO=-10
 
+USER="mpd"
+USERID="991"
+
 #TODO: Add uci config - nice, config
 
 start_service() {
 	local pld lport
 
+	user_exists $USER $USERID || user_add $USER $USERID
```

Comment:
NAK. This can be handled in the Makefile.

Reply:
Ah thanks, fixed. I've seen `user_exists` in many other packages as example :)

Reply:
One more question: do i have to hardcoded a specific user id like 991 or can it be automated. Some packages do not assign any user id.

Reply:
If you don’t include one it gets generated.